install/kubernetes/cilium/values.yaml

```diff
@@ -3074,6 +3074,14 @@ clustermesh:
       #               The "remote" certificate must be generated with CN=remote-<cluster-name>
       #               if provided manually. Cluster mode is meaningful only when the same
       #               CA is shared across all clusters part of the mesh.
+
+      # -- Allow users to provide their own certificates
+      # Users may need to provide their certificates using 
+      # a mechanism that requires they provide their own secrets.
+      # This setting does not apply to any of the auto-generated 
+      # mechanisms below, it only restricts the creation of secrets
+      # via the `tls-provided` templates.
+      enableSecrets: true
```

Comment:
I think this needs to be moved down below `authMode`. It's cutting out the docstring

Reply:
Fixed, thanks for pointing that out!